packages/server/routes/api/events.js

```diff
@@ -117,26 +117,23 @@ router.get(
 router.get(
   "/:eventId/attendees",
   AuthController.authenticate,
+  [
+    param("eventId")
+      .trim()
+      .blacklist("<>")
+      .isUUID()
+      .withMessage((eventId) => `Given eventId ${eventId} is invalid.`)
+      .bail()
+      .custom(async (eventId) => {
+        if (!(await EventController.existsInPrisma(eventId))) {
+          throw new Error(`No event exists with an ID of ${eventId}`);
+        }
+      }),
+  ],
```

Comment:
Since we will have many validators for the different routes, it would be best to put the validators in their own file so we don't clutter up the route handler files. The convention the rest of the team is doing is putting validators into a `packages/server/validators` folder and since you're doing events it'll be in `packages/server/validators/events.js`. Move this validator into that file and then export it to use here instead.